packaging/redhat/python-apprise.spec

```diff
@@ -51,11 +51,11 @@ Apprise API, AWS SES, AWS SNS, Boxcar, ClickSend, DAPNET, DingTalk, Discord, E-M
 Emby, Faast, FCM, Flock, Gitter, Google Chat, Gotify, Growl, Home Assistant,
 IFTTT, Join, Kavenegar, KODI, Kumulos, LaMetric, MacOSX, Mailgun, Mattermost,
 Matrix, Microsoft Windows, Microsoft Teams, MessageBird, MQTT, MSG91, MyAndroid,
-Nexmo, Nextcloud, NextcloudTalk, Notica, Notifico, Office365, OneSignal, Opsgenie,
-ParsePlatform, PopcornNotify, Prowl, Pushalot, PushBullet, Pushjet, Pushover,
-PushSafer, Reddit, Rocket.Chat, SendGrid, ServerChan, SimplePush, Sinch, Slack,
-SMTP2Go, Spontit, SparkPost, Super Toasty, Streamlabs, Stride, Syslog, Techulus Push,
-Telegram, Twilio, Twitter, Twist, XBMC, XMPP, Webex Teams}
+Nexmo, Nextcloud, NextcloudTalk, Notica, Notifico, Ntfy, Office365, OneSignal,
```

Comment:
Same thing about the name; as in many other places: "ntfy" not "Ntfy"

Reply:
Yep I knew that about ntfy, my *only* hangup here is that no other services are all lowercase -- including "E-Mail", oddly -- which makes me wonder if that's some sort of unspecified naming requirement here.

Reply:
I knew what I was getting into deciding to call it "ntfy". I have years of cringeperience with VMware, jQuery and others being misspelled. But I'll stick to the name :-D I'd say just update it and see what Chris says. :shrug: